examples/src/components/LineChartExample.jsx

```diff
@@ -28,13 +28,21 @@ export class LineChartExample extends Component {
 
     render() {
         const measures = [
-            Model.measure(franchiseFeesIdentifier).format("#,##0"),
-            Model.measure(franchiseFeesAdRoyaltyIdentifier).format("#,##0"),
-            Model.measure(franchiseFeesInitialFranchiseFeeIdentifier).format("#,##0"),
-            Model.measure(franchiseFeesIdentifierOngoingRoyalty).format("#,##0"),
+            Model.measure(franchiseFeesIdentifier)
+                .format("#,##0")
+                .localIdentifier("m0"),
```

Comment:
Can you please use something like `"franchiseFeesIdentifier"` like in other files?